lib/nailinda/accounts/patient.ex

```diff
@@ -29,5 +29,20 @@ defmodule Nailinda.Accounts.Patient do
       :location
     ])
     |> validate_required([:first_name, :last_name, :date_of_birth])
+    |> validate_date_is_not_future
+  end
+
+  defp validate_date_is_not_future(changeset) do
```

Comment:
Though not strict requirement, we like having private functions start with `_` .
This allows us to know whether its a private function defined within the module or its been imported from other modules

In this case we can have 
``` elixir 
defp _validate_date_is_not_future(changeset) do
......
end
```

Reply:
I have updated the PR.

use _ prefix for private function
squashed commits to a single commit.

Reply:
This is great @sandeshsoni :smiley:  I love your approach towards this. Will merge it in after the other approvals. Good job!